package.json

```diff
@@ -68,135 +68,108 @@
     "npm": ">= 5.3.0"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.10"
   },
   "peerDependencies": {
-    "@angular/core": "^4.4.6",
-    "@angular/http": "^4.4.6",
-    "ngx-base": "^2.3.0",
-    "rxjs": "^5.0.1"
+    "@angular/core": "^6.1.0-rc.0",
+    "@angular/http": "^6.1.0-rc.0",
+    "ngx-base": "^2.3.6",
+    "rxjs": "^6.2.1"
   },
   "devDependencies": {
-    "@angular/common": "^4.4.6",
-    "@angular/compiler": "^4.4.6",
-    "@angular/compiler-cli": "^4.4.6",
-    "@angular/core": "^4.4.6",
-    "@angular/forms": "^4.4.6",
-    "@angular/http": "^4.4.6",
-    "@angular/platform-browser": "^4.4.6",
-    "@angular/platform-browser-dynamic": "^4.4.6",
-    "@angular/router": "^4.4.6",
+    "@angular/common": "6.1.0-rc.0",
+    "@angular/compiler": "6.1.0-rc.0",
+    "@angular/compiler-cli": "6.1.0-rc.0",
+    "@angular/core": "6.1.0-rc.0",
+    "@angular/forms": "6.1.0-rc.0",
+    "@angular/http": "6.1.0-rc.0",
+    "@angular/platform-browser": "6.1.0-rc.0",
+    "@angular/platform-browser-dynamic": "6.1.0-rc.0",
+    "@angular/router": "6.1.0-rc.0",
     "@krux/condition-jenkins": "1.0.1",
-    "@types/core-js": "0.9.43",
-    "@types/jasmine": "2.6.2",
-    "@types/lodash": "4.14.80",
-    "angular2-load-children-loader": "0.2.1",
-    "angular2-router-loader": "0.3.5",
+    "@ngtools/webpack": "6.0.8",
+    "@types/core-js": "2.5.0",
+    "@types/jasmine": "2.8.8",
+    "@types/lodash": "4.14.112",
+    "ajv": "6.5.2",
```

Comment:
Why is this needed?

Reply:
It fixed a warning but now when I remove it there is no warning, hmm.
I'll take it out for now.